flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/sink/writer/AsyncSinkWriter.java

```diff
@@ -139,21 +171,40 @@ public AsyncSinkWriter(
         this.maxBatchSize = maxBatchSize;
         this.maxInFlightRequests = maxInFlightRequests;
         this.maxBufferedRequests = maxBufferedRequests;
+        this.flushOnBufferSizeMB = flushOnBufferSizeMB;
+        this.maxTimeInBufferMS = maxTimeInBufferMS;
```

Comment:
Can you add sanity validations for all the numeric inputs? Check for min/max etc